packages/block-editor/src/components/block-removal-warning-modal/index.js

```diff
@@ -41,6 +41,18 @@ export function BlockRemovalWarningModal( { rules } ) {
 		return;
 	}
 
+	const messages = {
+		patternOverrides: _n(
+			'Deleting this block could break patterns on your site that have content linked to it. Are you sure you want to delete it?',
+			'Deleting these blocks could break patterns on your site that have content linked to them. Are you sure you want to delete them?',
+			blockNamesForPrompt.length
+		),
+		templates: _n(
+			'Deleting this block will stop your post or page content from displaying on this template. It is not recommended.',
+			'Deleting these blocks will stop your post or page content from displaying on this template. It is not recommended.',
+			blockNamesForPrompt.length
+		),
+	};
```

Comment:
could just do a ternary here, but these seems easier to read with these long vars, and also easier to add more message types later this way.

Reply:
It's recommended to use ternaries in similar cases. See https://developer.wordpress.org/reference/functions/_n/#comment-2397.

Reply:
thanks, have switched back to a ternary, was probably a premature optimisation, we can easily change if we get a third option added.